doc/src/build/fullnode.md

```diff
@@ -288,3 +288,119 @@ If you followed the instructions for [Building from Source](#building-from-sourc
+by adding `num_epochs_to_retain_for_checkpoints` to their `authority-store-pruning-config` config:
+
+```yaml
+authority-store-pruning-config:
+  num-latest-epoch-dbs-to-retain: 3
+  epoch-db-pruning-period-secs: 3600
+  num-epochs-to-retain: 0
+  max-checkpoints-in-batch: 10
+  max-transactions-in-batch: 1000
+  # Number of latest epochs before which transaction and effects could be pruned.
+  # When this is N (where N >= 2), pruner can prune transaction and effects from 
+  # checkpoints upto current - N epoch. Transaction and effects from current and
+  # previous epoch are never pruned. N = 2 is a recommended setting for validators
+  # and fullnodes not serving rpc requests.
+  num_epochs_to_retain_for_checkpoints: 2
+  # Ensures that individual SST files periodically go through the compaction process.
+  # This helps reclaim disk space and avoid fragmentation issues
+  periodic-compaction-threshold-days: 1
+```
+
+## Archival Fallback
+
+Once various full nodes start pruning historical transactions and effects, it may not be always possible for
+their peers to catch up transaction and effects via state synchronization. And for this a node needs to fallback
+downloading this data from a remote aws s3 bucket. This archive has all transaction and effects data since 
+genesis and lags behind the latest network checkpoint by ten minutes. Enabling this is very highly recommended
+for all nodes. A sui node can enable this automatic archive fallback by adding the following config:
+
+```yaml
+state-archive-read-config:
+  - object-store-config:
+      object-store: "S3"
+      # Use mysten-testnet-archives for testnet 
+      # Use mysten-mainnet-archives for mainnet
+      bucket: "mysten-<testnet|mainnet>-archives"
+      # Use your aws account access key id
+      aws-access-key-id: "<AWS_ACCESS_KEY_ID>"
+      # Use your aws account secret access key
+      aws-secret-access-key: "<AWS_SECRET_ACCESS_KEY>"
+      aws-region: "us-west-2"
+      object-store-connection-limit: 20
+    # How many objects to read ahead when catching up  
+    concurrency: 5
+    # Whether to prune local state based on latest checkpoint in archive.
+    # This should stay false for most use cases
+    use-for-pruning-watermark: false
+```
+
+## Setting up your own archival fallback
+
+It is also possible to set up the archival fallback to a different (personal) s3 bucket. For this a fullnode
+needs to be configured to archive transactions and effects. Enabling this can be done by adding the below config:
+
+```yaml
+state-archive-write-config:
+  object-store-config:
+    object-store: "S3"
+    bucket: "<bucket_name>"
+    aws-access-key-id: "<AWS_ACCESS_KEY_ID>"
+    aws-secret-access-key: "<AWS_SECRET_ACCESS_KEY>"
+    aws-region: "<aws_region>"
+    object-store-connection-limit: 20
+  concurrency: 5
+  # This is needed to be set as true on the node which is archiving the data
```

Comment:
is it true or True?

on the node that archives the data

Reply:
it should be `true`